solr/contrib/ltr/src/test/org/apache/solr/ltr/rest/TestModelManager.java

```diff
@@ -20,7 +20,7 @@
 import org.apache.solr.common.util.NamedList;
 import org.apache.solr.core.SolrResourceLoader;
 import org.apache.solr.ltr.TestRerankBase;
-import org.apache.solr.ltr.ranking.LTRComponent;
+import org.apache.solr.ltr.ranking.LTRQParserPlugin;
```

Comment:
Verify the changes you added work in cloud mode with shards + replicas, and when restarting the instances they still pick up the existing deployment managed resources + Add some tests verifying that's the case.  Jon added a TestSolrCloud.java, which you can probably leverage to add more cases (if needed)